Gruntfile.js

```diff
@@ -137,7 +137,7 @@ module.exports = function( grunt ) {
 						"ascii_only": true
 					},
 					banner: "/*! jQuery v<%= pkg.version %> | " +
-						"(c) 2005, <%= grunt.template.today('yyyy') %> jQuery Foundation, Inc. | " +
+						"(c) jQuery Foundation, Inc. | " +
```

Comment:
We don't need them, according to lawyers. So removing the dates makes it easier to maintain! :smiley: